Update vault.md

MikeiLL opened the pull request:
Correct/update link

MikeiLL committed:
Update vault.md

Correct/update link

swalkinshaw commented on trellis/vault.md:
```diff
@@ -86,7 +86,7 @@ Run `trellis vault` to see usage details.
 ## Working with vault variables
 
-Here are a few tips for working with [variables and vault](http://docs.ansible.com/ansible/playbooks_best_practices.html#variables-and-vaults) in Trellis.
+Here are a few tips for working with [variables and vault](https://docs.ansible.com/ansible/latest/tips_tricks/ansible_tips_tricks.html#variables-and-vaults) in Trellis.
```
🤔 this link didn't work for me either.

Think it's now: https://docs.ansible.com/ansible/latest/tips_tricks/ansible_tips_tricks.html#keep-vaulted-variables-safely-visible

retlehs committed:
Update vault.md with new date and link correction

Updated the date_modified to 2026-03-10 and corrected the link for vault variables.

retlehs committed:
Merge branch 'docs' into patch-2
